webroot/css/scss/layout/_content.scss

```diff
@@ -408,6 +408,10 @@ strong {
                         .red-icon {
                             color: red;
                         }
+
+                        .blue-icon {
+                            color: #0096FF;
```

Comment:
can we use vars?

Reply:
This has been improved yesterday. Changes will be added soon